CONTRIBUTING.rst

```diff
@@ -35,5 +37,21 @@ documentation.
 * The `ACRN user mailing list`_ is a great place to engage with the
   community, ask questions, discuss issues, and help each other.
 
+.. _tsc_members:
+
+Technical Steering Committee (TSC)
+**********************************
+
+The Technical Steering Committee (TSC) is responsible for technical oversight of
```

Comment:
Wouldn't it  better to say "...responsible for all technical..." or "...responsible for the technical ..."?